feedbacks/tests/mixins.py

```diff
@@ -0,0 +1,40 @@
+from django.contrib.auth import get_user_model
+
+from feedbacks.models import Feedback, Recommendation
+
+__all__ = ['FeedbackCRUDViewsSetUpMixin', 'RecommendationCRUDViewsSetUpMixin',
+           'SetUpUserMixin', 'USERNAME', 'PASSWORD']
+
+USERNAME = 'test_user'
+PASSWORD = '12345'
+
+
+class SetUpUserMixin(object):
+
+    @classmethod
+    def setUpTestData(cls):
+        get_user_model().objects.create_user(username=USERNAME,
+                                             password=PASSWORD)
+
+
+class RecommendationCRUDViewsSetUpMixin(SetUpUserMixin):
+
+    @classmethod
+    def setUpTestData(cls):
+        super().setUpTestData()
+        Feedback.objects.create(feedback='Some text', min_value=0, max_value=10,
+                                indicator='PDI')
+
+    def setUp(self):
+        self.feedback_pk = Feedback.objects.get(pk=1).id
+        self.recommendation_pk = Recommendation.objects.create(
+            recommendation='Lorem ipsum', feedback_id=self.feedback_pk).id
+
+
+class FeedbackCRUDViewsSetUpMixin(SetUpUserMixin):
+
+    def setUp(self):
+        self.pk = Feedback.objects.create(feedback='Some text',
+                                                       min_value=0,
+                                                       max_value=10,
+                                                       indicator='PDI').id
```

Comment:
https://github.com/IF-093-Python/CultureAnalyzer/blob/0992c02e7c32fc1ef1a5697928963be0b70b9271/feedbacks/tests/mixins.py#L25-L26

It's better to would be:
``` python
Feedback.objects.create(feedback='Some text', indicator='PDI',
                        min_value=0, max_value=10)
```
PEP8: continuation line over-indented for visual indent 

https://github.com/IF-093-Python/CultureAnalyzer/blob/0992c02e7c32fc1ef1a5697928963be0b70b9271/feedbacks/tests/mixins.py#L37-L40

``` python 
self.pk = Feedback.objects.create(feedback='Some text',
                                  min_value=0,
                                  max_value=10,
                                  indicator='PDI').id
```